spec/money_column_spec.rb

```diff
@@ -91,6 +97,21 @@ class MoneyRecordCoerceNull < ActiveRecord::Base
     expect(record.prix.currency.to_s).to eq('CAD')
   end
 
+  it 'does not overwrite a currency column with a null currency' do
+    Money.with_currency(Money::NullCurrency.new) do
```

Comment:
I can remove this `with_currency` here once https://github.com/Shopify/money/pull/102 is merged